libbeat/processors/config.go

```diff
@@ -9,22 +9,22 @@ import (
 )
 
 type ConditionConfig struct {
-	Equals   *ConditionFilter `config:"equals"`
-	Contains *ConditionFilter `config:"contains"`
-	Regexp   *ConditionFilter `config:"regexp"`
-	Range    *ConditionFilter `config:"range"`
+	Equals   *ConditionFields `config:"equals"`
+	Contains *ConditionFields `config:"contains"`
+	Regexp   *ConditionFields `config:"regexp"`
+	Range    *ConditionFields `config:"range"`
 }
 
-type ConditionFilter struct {
+type ConditionFields struct {
 	fields map[string]interface{}
 }
 
-type FilterPluginConfig []map[string]common.Config
+type ProcessorPluginConfig []map[string]common.Config
```

Comment:
let's rename to 'PluginConfig' as the package name is always part of the type name in golang.